prebid-mobile/amp-creative.md

```diff
@@ -0,0 +1,16 @@
+<script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@0.4.0/dist/creative.js"></script>
```

Comment:
Please remove this file and update the one in _includes/dev-docs/amp-creative.md

 Remove the spaces in 'src = "https' to 'src="https' -- otherwise the highlighting is a little off.

And please add appropriate indenting below.